### DISCUSSION TITLE
Font issue with Rendered Quarto HTML

### DISCUSSION THREAD

**Discussion by jmgirard** [quarto-dev/quarto-cli#4208] [Q&A]
I have a student on a Windows 11 laptop that is running into issues with the font display of his rendered Quarto HTML documents. The font is wrong (and quite ugly) in both the Viewer pane and when bumped out into browser. Weirdly enough, if he sends the file to me, it displays with the proper font. He had the same kind of issue with Rmarkdown HTML but then I reinstalled "sass" package and that fixed it there (but not for Quarto). Any ideas what we need to do to fix this? Reinstall some other package perhaps?

![Screenshot (27)](https://user-images.githubusercontent.com/6829033/216683288-7a6f130e-67ed-4bfc-8058-bd9fbd9377d2.png)

**Comment by cscheid** [comment]
It looks like their monospaced font in the default OS theme is the thing causing it. I'd recommend them to open a web browser and inspect the computed properties of the element, by right clicking on the output that has a bad font, clicking "inspect", and then clicking on the "Computed" tab on the side (at least this is how you'd do it in Chrome. Since I see Chrome on their launch bar or whatever-the-windows-name-is, that's what I'd recommend)

<img width="1274" alt="image" src="https://user-images.githubusercontent.com/285675/216782475-c6559db0-e1f5-40dc-ad3d-07f8d1c86fb6.png">
